lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -676,6 +679,127 @@ def set_communication(self, trans, id, payload={}, **kwd):
         trans.sa_session.flush()
         return {'message': message}
 
+    @expose_api
+    def get_custom_builds_metadata(self, trans, id, payload={}, **kwd):
+        """ Returns meta data for custom builds. """
```

Comment:
Can you please add some documentation to the new API calls (like the one `create()`, `update()` and `delete()` methods have)?